src/ApiPort.VisualStudio/AnalyzeMenu.cs

```diff
@@ -171,5 +184,23 @@ private ICollection<Project> GetSelectedProjects()
                 .Distinct()
                 .ToList();
         }
+
+        private ICollection<Project> GetTransitiveReferences(IEnumerable<Project> projects, HashSet<Project> expandedProjects = null)
```

Comment:
Look a couple of lines down. Since this is recursive, the initial call doesn't need to supply this, but it is checked the second line down. I had originally had the caller supply a the hashset - I'll switch it back to that so it's clearer.